tests/components/nordpool/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""Tests for the Nord Pool integration."""
+
+from homeassistant.components.nordpool.const import CONF_AREAS
+from homeassistant.const import CONF_CURRENCY
+
+ENTRY_CONFIG = {
+    CONF_AREAS: ["SE3", "SE4"],
+    CONF_CURRENCY: "SEK",
+}
```

Comment:
This looks like something for `const.py`

Reply:
Have never seen a `const.py` file for tests. Seems quite unnecessary?